keel-core/src/main/java/com/netflix/rocket/api/artifact/internal/debian/DebianArtifactParser.java

```diff
@@ -0,0 +1,57 @@
+package com.netflix.rocket.api.artifact.internal.debian;
```

Comment:
Any particular reason to name this `rocket` and not `spinnaker.keel`?

Reply:
It's a direct lift from their code, same as the DebianComparator. Just followed that pattern